src/index.ts

```diff
@@ -10,6 +10,8 @@ export { default as FromQuery } from "./decorators/FromQuery"
 export { default as Router } from "./Router"
 export { default as RequestError } from "./RequestError"
 export { default as RouterError } from "./RouterError"
-export * as constants from "./constants";
+export { default as RouteMap } from "./RouteMap"
+export { createLambdaContext, createAPIGatewayProxyEvent } from "../tests/testUtil"
```

Comment:
wondering if it's a good idea to expose these test functions? These are really internals, it might be confusing to expose them like this.

Reply:
Yeah that's true, i'm repeating those Event Mocks on a lambda i have, wonder if there' s abetter way to make them available otherwise i'll just copy them

Reply:
maybe just a vacasa lambaa testing library

Reply:
juts removed that export @jackholmes1010